adafruit_circuitplayground/express.py

```diff
@@ -63,6 +66,10 @@ def __init__(self):
         self._temp = adafruit_thermistor.Thermistor(board.TEMPERATURE, 10000, 10000, 25, 3950)
         self._light = Photocell(board.LIGHT)
 
+        # Define audio:
+        self.speaker_enable = digitalio.DigitalInOut(board.SPEAKER_ENABLE)
+        self.speaker_enable.switch_to_output(value=True)
```

Comment:
Lets not set the value to True until we want to play sound. Otherwise the speaker will crackle randomly.